config/app/init/base/init.json

```diff
@@ -14,12 +15,41 @@
       "@type": "SequenceHandler",
       "handlers": [
         { "@id": "urn:solid-server:default:LoggerInitializer" },
-        { "@id": "urn:solid-server:default:BaseUrlVerifier" },
-        { "@id": "urn:solid-server:default:ParallelInitializer" },
-        { "@id": "urn:solid-server:default:SeededPodInitializer" },
-        { "@id": "urn:solid-server:default:ServerInitializer" },
-        { "@id": "urn:solid-server:default:ModuleVersionVerifier" }
+        { "@id": "urn:solid-server:default:PrimaryInitializer" },
+        { "@id": "urn:solid-server:default:WorkerInitializer" }
       ]
+    },
+    {
+      "comment": "This wrapped sequence handler will be executed ONLY BY THE PRIMARY PROCESS when starting the server.",
+      "@id": "urn:solid-server:default:PrimaryInitializer",
+      "@type": "PrimaryHandler",
+      "source": {
+        "comment": "These initializers will all be executed sequentially when starting the server.",
+        "@id": "urn:solid-server:default:PrimarySequenceInitializer",
+        "@type":"SequenceHandler",
+        "handlers": [
+          { "@id": "urn:solid-server:default:BaseUrlVerifier" },
+          { "@id": "urn:solid-server:default:ParallelInitializer" },
```

Comment:
The reason this initializer was made is so that users could easily add an initializer to the list of initializers (since new initializers still had to appear before the `ServerInitializer`). This is still relevant here because of the `WorkerManager`, but we should probably rename it to `PrimaryParallelInitializer` and create a similar one for the workers and update the init configs accordingly.

Reply:
Ok, but then for instance in `config/app/init/initializers/default.json`, I would probably not only want to change the name to `PrimaryParallelInitialilizer` but also add an extra `WorkerParallelInitializer` with correct comments, allowing the user to choose what they want?

Reply:
Yes, a new one would have to be added (with an empty array) in all of those configs.